### PR TITLE
Update Logger.java

### PR DESCRIPTION
**Pull request by tanmaymanolkar1**
unblock the release, done to keep the gradle changes in check

### CODE REVIEW

**Review by gitaumoses4 — APPROVED** [review]

File: common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java [hunk 1]
```diff
@@ -149,10 +150,20 @@ public void setExternalLogger(final ILoggerCallback externalLogger) {
      * @return String The concatenation of thread_name and correlation_id to serve as the required metadata in the log lines.
      */
     public static String getDiagnosticContextMetadata() {
+        String threadName = DiagnosticContext.getRequestContext().get(DiagnosticContext.THREAD_NAME);
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java:153]:
```suggestion
        final String threadName = DiagnosticContext.getRequestContext().get(DiagnosticContext.THREAD_NAME);
```

File: common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java [hunk 2]
```diff
@@ -149,10 +150,20 @@ public void setExternalLogger(final ILoggerCallback externalLogger) {
...
+        String correlationId = DiagnosticContext.getRequestContext().get(DiagnosticContext.CORRELATION_ID);
+
+        if(StringUtil.isEmpty(threadName)){
+            threadName = "UNSET";
+        }
+        if(StringUtil.isEmpty(correlationId)){
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java:159]:
```suggestion

        if (StringUtil.isEmpty(correlationId)){
```
Formatting

File: common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java [hunk 3]
```diff
@@ -149,10 +150,20 @@ public void setExternalLogger(final ILoggerCallback externalLogger) {
      * @return String The concatenation of thread_name and correlation_id to serve as the required metadata in the log lines.
      */
     public static String getDiagnosticContextMetadata() {
+        String threadName = DiagnosticContext.getRequestContext().get(DiagnosticContext.THREAD_NAME);
+        String correlationId = DiagnosticContext.getRequestContext().get(DiagnosticContext.CORRELATION_ID);
+
+        if(StringUtil.isEmpty(threadName)){
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java:156]:
```suggestion
        if (StringUtil.isEmpty(threadName)){
```

(Consider using `ctrl` + `alt` + `l` to auto format code for consistency)

**Review by iambmelt — APPROVED** [review]
Approving with formatting suggestions

File: common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java (hunk 3 above)
Comment (tanmaymanolkar1) [reply common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java]: addressed, thanks :)

**Pull request merged**